website/source/docs/guides/common-errors.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: "docs"
+page_title: "Common Error Messages"
+sidebar_current: "docs-guides-common-errors"
```

Comment:
Since the guides section will be removed later this month, this should be located some where else. 
I think we have two options:
1. The "Agent" category (at the bottom). This doesn't seem to fit into any other category. This location wouldn't alter the side nav. 
2. Its own "Common Errors" category above the "FAQ" (kind of in the existing guides location).

Long-term, I am not sure this should be a side navigation category. However, getting data from Google Analytics could help us decided on a more permanent location for the upcoming reorg.

Reply:
Personally, I like above the FAQ. It can be part of the "I have problems" section 😆